node/src/components/network.rs

```diff
@@ -397,17 +398,17 @@ impl<REv: ReactorEventT<P>, P: PayloadT> Network<REv, P> {
             );
         }
 
-        for &peer_id in &peer_ids {
-            self.send_message(peer_id.clone(), payload.clone());
+        for &&peer_id in &peer_ids {
```

Comment:
Nit: odd usage of && - if I understand correctly `peer_ids` could be `.copied()` before `.filter()` and then we can have `for peer_id in peer_ids`  (or `for &peer_id in &peer_ids`)